File: Assets/PlayerTraits.js
#pragma strict
public var CreatePlane : CreatePlane;


function Start () {
var CreatePlane = GetComponent("CreatePlane");

}

function OnCollisionEnter ( collider : Collision) {
	if( collider.gameObject.tag == "Obstacle") {
		transform.position = new Vector3(Random.Range(- CreatePlane.width, CreatePlane.width), 0.1, Random.Range(- CreatePlane.height, CreatePlane.height));
	Debug.Log("Hitting");
	}
}

function Update () {

}

File: Assets/Prefabs/PlayerTraits.cs
using UnityEngine;
using System.Collections;

public class PlayerTraits : MonoBehaviour {

	public CreatePlane CreatePlane;
	public GameObject node;
	public bool collision= false;


	void  Start (){
		CreatePlane = GetComponent<CreatePlane>();
		GameObject node = this.gameObject;
	}

}